aidants_connect_habilitation/templates/email/issuer_profile_reminder.html

```diff
@@ -15,12 +15,13 @@
                       <p>Bonne nouvelle ! Il existe déjà un espace demandeur associé à cet email chez nous.</p>
                       <p>
                         Vous pouvez <a href="{{ url }}">suivre le lien suivant pour retrouver toutes vos demandes en
-                        cours et en déposer de nouvelles</a> ou, si cela ne fonctionne pas, en copier l'URL suivante
+                        cours et en déposer de nouvelles</a> ou, si cela ne fonctionne pas, en copiant l'URL suivante
```

Comment:
```suggestion
                        cours et en déposer de nouvelles</a> ou, si cela ne fonctionne pas, en copiant l’URL suivante
```

(Apostrophe typographique)